testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/TestClassProvider.java

```diff
@@ -0,0 +1,82 @@
+package org.keycloak.testsuite.arquillian;
+
+import com.sun.net.httpserver.HttpContext;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpServer;
+import org.apache.commons.io.IOUtils;
+import org.jboss.logging.Logger;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
+
+public class TestClassProvider {
+
+    private static final Logger LOGGER = Logger.getLogger(TestClassProvider.class);
+
+    public static final String[] PERMITTED_PACKAGES = new String[] {
+            "/org/keycloak/testsuite",
+            "/org/junit",
+            "/org/hamcrest",
+            "/org/keycloak/admin/client",
+            "/org/jboss/resteasy/client",
+            "/org/jboss/arquillian",
+            "/org/jboss/shrinkwrap",
+            "/org/openqa/selenium"
+    };
```

Comment:
Nit: This array could be then turned into a regexp to and use single `Pattern.matches()` instead of iteration within `isPermittedPackage` method

Reply:
I prefer it as is